rancid-tomatillos/src/components/Header.js

```diff
@@ -1,9 +1,25 @@
-import React from 'react'
-
+import React from "react";
+import {BrowserRouter as Router, Switch, Route, Link} from "react-router-dom";
 function Header() {
     return (
-        <h1>Rotten Tomatillos</h1>
-    )
+        <nav>
+            <h1>Rotten Tomatillos</h1>
+            <ul>
+                <li>
+                    <Link to="/">Home</Link>
+                </li>
+                <li>
+                    <Link to="/about">About</Link>
+                </li>
+                <li>
+                    <Link to="/users">Users</Link>
+                </li>
+                <li>
+                    <Link to="/login">Login</Link>
+                </li>
+            </ul>
+        </nav>
+    );
```

Comment:
Some of these will probably not be links, may be able to reduce this to just the login / logout functionality. Just for getting router up and working for now

Reply:
Sounds good! "Home" might still be handy for when a user wants to go from individual movie detail page back to main page but "About" is likely not necessary as of yet